stdlib/logging/config.pyi

```diff
@@ -106,7 +106,7 @@ class ConvertingTuple(tuple[Any, ...], ConvertingMixin):  # undocumented
     @overload
     def __getitem__(self, key: slice) -> Any: ...
 
-class BaseConfigurator:  # undocumented
```

Comment:
`BaseConfigurator` is mentioned a little bit in the [*Import resolution and custom importers* part of the docs]( https://docs.python.org/3/library/logging.config.html#import-resolution-and-custom-importers)